### PR TITLE
Chi 1186 ux design debt sampler

### PR DESCRIPTION
**Pull request by acedeywin**
Primary reviewer: @stephenhand @nickhurlburt 

### Checklist
- [n/a] Corresponding issue has been opened
- [n/a] New tests added
- [n/a] Strings are localized
- [n/a] Tested for chat contacts
- [n/a] Tested for call contacts

### Related Issues
Fixes # N/A

### CODE REVIEW

**Review by nickhurlburt — COMMENTED** [review]
Translation edits look good. One edit to contacts waiting, and please roll back package-lock changes.

File: plugin-hrm-form/src/components/queuesStatus/QueueCard.tsx [hunk 1]
```diff
@@ -28,7 +28,7 @@ const getChannelUI = (
       className="channel-box-inner-value"
       data-testid="channel-box-inner-value"
     >
-      {value}
+      {value === 0 ? '-' : value}
```
[the remark this answers is not in the document]
Comment (acedeywin) [reply plugin-hrm-form/src/components/queuesStatus/QueueCard.tsx]: It's done. Since it's an HTML symbol, I had to add it in a `<span>` tag.

**Comment by acedeywin** [comment]
@stephenhand all tests are passing.

**Review by stephenhand — APPROVED** [review]
I think it's ok this time, since you were having issues with NPM versions etc, but going forward we would only normally update the package-lock.json files when we update the package.json or the flex version.

But LGTM

**Pull request merged**